libmamba/tests/src/core/test_activation.cpp

```diff
@@ -16,5 +16,24 @@ namespace mamba
             // std::endl; std::cout << a.activate("/home/wolfv/miniconda3/", false) <<
             // std::endl;
         }
+
+        TEST_CASE("Activator::get_default_env")
+        {
+            ContextOptions opts;
+            Context ctx(opts);
```

Comment:
No need to explicitly pass options when empty, [it's already the case by default](https://github.com/mamba-org/mamba/blob/main/libmamba/include/mamba/core/context.hpp#L262).